pkg/controller/install/certresources.go

```diff
@@ -157,6 +158,13 @@ func ServiceName(deploymentName string) string {
 	return deploymentName + "-service"
 }
 
+func HostnamesForService(serviceName, namespace string) []string {
+	return []string{
+		fmt.Sprintf("%s.%s", serviceName, namespace),
+		fmt.Sprintf("%s.%s.svc", serviceName, namespace),
```

Comment:
not adding `.cluster` nor `.cluster.local`?

Reply:
These should resolve even without the cluster domain specified, though we may need to test for the current cluster domain before adding `.cluster` or `.cluster.local` in case it's different from the default.